pkg/jx/cmd/get_apps.go

```diff
@@ -117,7 +190,31 @@ func (o *GetAppsOptions) generateTable(apps *v1.AppList, kubeClient kubernetes.I
 func (o *GetAppsOptions) generateTableHeaders(apps *v1.AppList) table.Table {
 	t := o.createTable()
 	t.Out = o.CommonOptions.Out
-	titles := []string{"Name", "Version", "Description", "Chart Repository"}
+	titles := []string{"Name", "Version", "Chart Repository", "Namespace", "Status", "Description"}
 	t.AddRow(titles...)
 	return t
 }
+
+func (o *GetAppsOptions) printHelmResourcesWithFormat(helmOutputJSON string) error {
+	h := HelmOutput{}
+	err := json.Unmarshal([]byte(helmOutputJSON), &h)
+	if err != nil {
+		return err
+	}
+	if o.Output == "" {
+		fmt.Fprintln(o.Out, h.helmInfoStatus.Resources)
+		return nil
+	}
+	return o.renderResult(h, o.Output)
+
+}
+
+func checkDifferentAppsInAppsSlice(apps *v1.AppList) bool {
+	x, a := apps.Items[0], apps.Items[1:]
```

Comment:
If `Items` has size 0, this will panic.

Reply:
Following the normal flow of execution, the apps.items slice getting to this not exported method should never be empty.